### ISSUE TITLE
`gulp-grunt` depends on a globally installed `grunt-cli`.

### ISSUE THREAD

**Issue by lolmaus** [gratimax/gulp-grunt#19]
Hi!

Here are some extracts from my `gulpfile.coffee` and `package.json`:

```coffee
gulp.task "svg2string", ->
  gulp.start "grunt-svg2string"
```

```
    "grunt-svg2string": "^0.1.6",
    "gulp-grunt": "^0.5.2",
```

When i ran `gulp svg2string`, i received this cryptic error:

```
$ gulp svg2string
[16:57:15] Requiring external module coffee-script/register
[16:57:18] Using gulpfile ~/Code/hb_frontend_angular/gulpfile.coffee
[16:57:18] Starting 'svg2string'...
[16:57:18] Starting 'grunt-svg2string'...
[16:57:18] Finished 'svg2string' after 5.56 ms

events.js:72
        throw er; // Unhandled 'error' event
              ^
Error: spawn ENOENT
  at errnoException (child_process.js:1011:11)
  at Process.ChildProcess._handle.onexit (child_process.js:802:34)
```

Lack of any indication of the reason left us clueless with the cause of the problem and possible solutions. There wasn't even an error message to google for.

While debugging this issue we decided to try running `grunt svg2string`. To be able to do that, i had to install Grunt CLI: `npm install -g grunt-cli`. After that, the command `gulp svg2string` started working fine without issue!

Thus we discovered by luck that `gulp-grunt` depends on a globally installed `grunt-cli`.

This should not be so. I expect my Gulp recipes to work after doing

    npm install -g gulp
    npm install
    gulp some-grunt-task

**Comment by gratimax** [comment]
This does seem like a true issue. I'll look into using the grunt library directly to run tasks, because so far running the command has been a shortcut.

**Comment by robatron** [comment]
:+1: 

Just ran into the exact same issue. Perhaps a simple error check and clear error message if `grunt` is available would suffice for now.

**Comment by simonsmith** [comment]
Have found the same issue here too. Would be great to get a fix :)

**Comment by gratimax** [comment]
Fixed by #22, courtesy of @Warbring3r
